### DISCUSSION TITLE
Getting really scared about the lack of SSR Auth for supabase

### DISCUSSION THREAD

**Discussion by CropWatchDevelopment** [supabase/supabase#21150] [Questions]
I NEED to have a project working with auth.
I need to get SSR Auth working, but there seem to be NO examples and conflicting docs for Sveltekit.
Does anyone know of any form of example projects for auth with Supabase?
This does work, but is the old login method: https://supabase.com/docs/guides/auth/auth-helpers/sveltekit?language=ts
This guide fails to work: https://supabase.com/docs/guides/auth/server-side/creating-a-client?environment=hooks

**Comment by J0** [comment]
Hey @CropWatchDevelopment 

Thanks for the feedback - was wondering if you could share more with your issues on: This guide fails to work: https://supabase.com/docs/guides/auth/server-side/creating-a-client?environment=hooks ?

Let us know!
